### PR TITLE
new icon: fedora (original, plain)

### PR DESCRIPTION
**Pull request by kaetaen**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

[Fedora Project](https://getfedora.org/pt_BR/)
[Fedora original icon](https://commons.wikimedia.org/wiki/File:Fedora_logo.svg)
[Fedora plain icon](https://commons.wikimedia.org/wiki/File:Fedora_icon_(2021).svg)

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Your images are not `128 x 128`. Please adjust as needed. You can use the [`iloveimg`](https://www.iloveimg.com/resize-image/resize-svg) tool which can help you resize your images. Let us know if you need any help! 😄

**Comment by kaetaen** [comment]
> Hey there! Your images are not `128 x 128`. Please adjust as needed. You can use the [`iloveimg`](https://www.iloveimg.com/resize-image/resize-svg) tool which can help you resize your images. Let us know if you need any help! smile

Oops.. obrigado pelo aviso.
Agora eu corrigi  a resolução dos ícones e otimizei com o SVGO

File: icons/fedora/fedora-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0 26.408-21.438 47.846-47.846 47.846-26.408 0-47.846-21.43-47.846-47.846 0-26.408 21.438-47.838 47.846-47.838Zm0 0" transform="rotate(180 59.856 68.144) scale(.4794)"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M66.926 61.137v29.89c0 16.54-13.41 29.953-29.95 29.953-2.511 0-4.296-.285-6.617-.89-3.39-.887-6.156-3.664-6.16-6.895 0-3.906 2.836-6.746 7.074-6.746 2.016 0 2.747.387 5.704.387 8.718 0 15.793-7.063 15.808-15.785V77.312c0-1.23-1-2.23-2.234-2.226l-10.387-.004c-3.867 0-6.996-3.086-6.996-6.965 0-3.906 3.16-6.98 7.07-6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M52.785 75.148V45.262c0-16.543 13.41-29.953 29.953-29.953 2.508 0 4.293.28 6.617.89 3.387.887 6.157 3.664 6.157 6.895 0 3.906-2.836 6.746-7.07 6.746-2.02 0-2.75-.387-5.704-.387-8.722 0-15.797 7.063-15.812 15.781v13.743a2.235 2.235 0 0 0 2.234 2.226l10.387.004c3.871 0 6.996 3.086 6.996 6.965.004 3.906-3.16 6.98-7.07 6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#3c6eb4;fill-opacity:1" d="M116.809 116.773v-2.652l-1.211 2.781-1.18-2.78v2.651h-.68v-4.187h.711l1.168 2.676 1.149-2.676h.722v4.187Zm-4.954-3.484v3.484h-.71v-3.484h-1.192v-.703h3.09v.703"/></svg>
```
Comment (Panquesito7) [inline icons/fedora/fedora-original.svg:1]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#294172;fill-opacity:1" d="M127.82 64.004C127.82 28.754 99.246.18 64 .18 28.766.18.203 28.73.18 63.957v49.39c.02 7.997 6.504 14.473 14.508 14.473h49.335c35.239-.015 63.797-28.578 63.797-63.816"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.123 142.096h47.846v47.847c0 26.408-21.43 47.846-47.846 47.846-26.409 0-47.838-21.438-47.838-47.846 0-26.409 21.43-47.847 47.838-47.847Zm0 0" transform="scale(.4794)"/><path style="fill:none;stroke-width:29.21;stroke-linecap:butt;stroke-linejoin:miter;stroke:#3c6eb4;stroke-opacity:1;stroke-miterlimit:4" d="M77.125 142.102h47.846v47.838c0 26.408-21.438 47.846-47.846 47.846-26.408 0-47.846-21.43-47.846-47.846 0-26.408 21.438-47.838 47.846-47.838Zm0 0" transform="rotate(180 59.856 68.144) scale(.4794)"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M66.926 61.137v29.89c0 16.54-13.41 29.953-29.95 29.953-2.511 0-4.296-.285-6.617-.89-3.39-.887-6.156-3.664-6.16-6.895 0-3.906 2.836-6.746 7.074-6.746 2.016 0 2.747.387 5.704.387 8.718 0 15.793-7.063 15.808-15.785V77.312c0-1.23-1-2.23-2.234-2.226l-10.387-.004c-3.867 0-6.996-3.086-6.996-6.965 0-3.906 3.16-6.98 7.07-6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#fff;fill-opacity:1" d="M52.785 75.148V45.262c0-16.543 13.41-29.953 29.953-29.953 2.508 0 4.293.28 6.617.89 3.387.887 6.157 3.664 6.157 6.895 0 3.906-2.836 6.746-7.07 6.746-2.02 0-2.75-.387-5.704-.387-8.722 0-15.797 7.063-15.812 15.781v13.743a2.235 2.235 0 0 0 2.234 2.226l10.387.004c3.871 0 6.996 3.086 6.996 6.965.004 3.906-3.16 6.98-7.07 6.98"/><path style="stroke:none;fill-rule:nonzero;fill:#3c6eb4;fill-opacity:1" d="M116.809 116.773v-2.652l-1.211 2.781-1.18-2.78v2.651h-.68v-4.187h.711l1.168 2.676 1.149-2.676h.722v4.187Zm-4.954-3.484v3.484h-.71v-3.484h-1.192v-.703h3.09v.703"/></svg>

```

File: icons/fedora/fedora-plain.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#51a2da;fill-opacity:1" d="M64.219 1.266C29.605 1.266 1.563 29.434 1.5 64.207h-.012v48.723H1.5c.016 7.886 6.383 14.273 14.242 14.273h48.465c34.621-.023 62.656-28.187 62.656-62.945 0-34.774-28.054-62.942-62.718-62.942Zm12.738 25.918c10.527 0 20.465 8.093 20.465 19.261 0 1.035.008 2.07-.164 3.246-.29 2.97-2.992 5.106-5.938 4.688-2.941-.426-4.941-3.23-4.394-6.168.05-.336.066-.863.066-1.766 0-6.332-5.156-8.773-10.031-8.773-4.875 0-9.27 4.117-9.277 8.773.082 5.387 0 10.727 0 16.114l9.05-.07c7.067-.145 7.149 10.538.082 10.488l-9.125.066c-.023 4.336.032 3.55.012 5.734 0 0 .074 5.297-.082 9.313-1.094 11.8-11.094 21.23-23.113 21.23-12.746 0-23.242-10.457-23.242-23.28.382-13.188 10.859-23.567 24.05-23.446l7.36-.055v10.469l-7.36.066h-.035c-7.25.215-13.465 5.16-13.582 12.957a12.765 12.765 0 0 0 12.805 12.797c7.086 0 12.758-5.18 12.758-12.781l-.012-39.649c.004-.738.027-1.324.11-2.132 1.195-9.7 9.831-17.09 19.609-17.09Zm0 0"/></svg>
```
Comment (Panquesito7) [inline icons/fedora/fedora-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#51a2da;fill-opacity:1" d="M64.219 1.266C29.605 1.266 1.563 29.434 1.5 64.207h-.012v48.723H1.5c.016 7.886 6.383 14.273 14.242 14.273h48.465c34.621-.023 62.656-28.187 62.656-62.945 0-34.774-28.054-62.942-62.718-62.942Zm12.738 25.918c10.527 0 20.465 8.093 20.465 19.261 0 1.035.008 2.07-.164 3.246-.29 2.97-2.992 5.106-5.938 4.688-2.941-.426-4.941-3.23-4.394-6.168.05-.336.066-.863.066-1.766 0-6.332-5.156-8.773-10.031-8.773-4.875 0-9.27 4.117-9.277 8.773.082 5.387 0 10.727 0 16.114l9.05-.07c7.067-.145 7.149 10.538.082 10.488l-9.125.066c-.023 4.336.032 3.55.012 5.734 0 0 .074 5.297-.082 9.313-1.094 11.8-11.094 21.23-23.113 21.23-12.746 0-23.242-10.457-23.242-23.28.382-13.188 10.859-23.567 24.05-23.446l7.36-.055v10.469l-7.36.066h-.035c-7.25.215-13.465 5.16-13.582 12.957a12.765 12.765 0 0 0 12.805 12.797c7.086 0 12.758-5.18 12.758-12.781l-.012-39.649c.004-.738.027-1.324.11-2.132 1.195-9.7 9.831-17.09 19.609-17.09Zm0 0"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Almost there! 😄

**Review by Thomas-Boi — APPROVED** [review]
This looks good now @kaetaen! Thank you for contributing.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Thomas-Boi — APPROVED** [review]

**Pull request merged**